simde/arm/neon/cvt.h

```diff
@@ -144,11 +144,14 @@ int16_t
 simde_vcvth_s16_f16(simde_float16 a) {
   #if defined(SIMDE_ARM_NEON_A64V8_NATIVE) && defined(SIMDE_ARM_NEON_FP16)
     return vcvth_s16_f16(a);
+  #elif defined(SIMDE_FAST_CONVERSION_RANGE)
+    return HEDLEY_STATIC_CAST(int16_t,
+        simde_float16_to_float32(a));
```

Comment:
There should be test cases added to `test_simde_vcvth_s16_f16` that show the different behavior if `SIMDE_FAST_CONVERSION_RANGE` is defined, or not.